app/src/test/java/org/simple/clinic/medicalhistory/newentry/NewMedicalHistoryUiRendererTest.kt

```diff
@@ -135,6 +136,50 @@ class NewMedicalHistoryUiRendererTest {
     verifyNoMoreInteractions(ui)
   }
 
+  @Test
+  fun `when patient has hypertension and country is india, then show hypertension treatment question`() {
+    // given
+    val model = defaultModel
+        .currentFacilityLoaded(facilityWithDiabetesManagementEnabled)
+        .answerChanged(DIAGNOSED_WITH_HYPERTENSION, Yes)
+
+    // when
+    uiRenderer.render(model)
+
+    // then
+    verifyImplicitRenders()
+    verify(ui).showDiagnosisView()
+    verify(ui).hideDiabetesHistorySection()
+    verify(ui).renderDiagnosisAnswer(DIAGNOSED_WITH_HYPERTENSION, Yes)
+    verify(ui).renderDiagnosisAnswer(DIAGNOSED_WITH_DIABETES, Unanswered)
+    verify(ui).showDiagnosisRequiredError(false)
+    verify(ui).hideNextButtonProgress()
+    verify(ui).showHypertensionTreatmentQuestion(Unanswered)
+    verifyNoMoreInteractions(ui)
+  }
+
+  @Test
+  fun `when patient has hypertension and country is not from india, then show hypertension treatment question`() {
+    // given
+    val uiRenderer = NewMedicalHistoryUiRenderer(ui, TestData.country(isoCountryCode = Country.BANGLADESH))
```

Comment:
Needing to create a different UI renderer for this specific case is a symptom of the problem I mentioned earlier, that this decision should be a business logic decision and not a UI renderer decision.